### ISSUE TITLE
No such iamge "apache/eventmesh-dashboard" on docker hub?

### ISSUE THREAD

**Issue by ZhengLin-Li** [apache/eventmesh-dashboard#44]
There may be an error in the docs of https://github.com/apache/eventmesh-dashboard?tab=readme-ov-file#getting-started-with-docker.

It seems that there is no such iamge "apache/eventmesh-dashboard" on docker hub?

<img width="1340" alt="image" src="https://github.com/apache/eventmesh-dashboard/assets/63448884/b051d1a1-8b37-4d70-8ef5-1a688de76a88">

**Comment by Pil0tXia** [comment]
Yes, the eventmesh-dashboard image haven't been uploaded to docker hub. You may have to use a self-built image or the image built by me: `pil0txia/eventmesh-dashboard:latest`.
